README.md

```diff
@@ -1,33 +1,28 @@
 
 Key features of the Radius platform include: 
-
-- *Radius Application graph*: Developers can model their entire application, including the connections and dependencies between services. 
-- *Swappable infrastructure*: Radius resources are platform-agnostic, allowing applications to be written once and deployed to any platform, such as Kubernetes, Microsoft Azure, Amazon Web Services (AWS), or on-premises hardware. 
-- Recipes for infrastructure Provisioning: Radius Recipes automate infrastructure provisioning using infrastructure-as-code templates, ensuring all deployments meet organizational requirements and policies. 
-- *Radius Environments*: IT operators can define policies, security configurations, Recipes, and diagnostics required by their organization, ensuring secure and repeatable infrastructure across local development, testing, and production environments. 
-- *Consistent tooling and experiences*: Radius offers a consistent set of APIs, tools, and experiences that span any cloud or on-premises environment, simplifying and standardizing cloud-native application teams' toolchains and processes.
+- *Team Collaboration*: Radius Applications and Environments allow developers to work with Operations on application definition and delivery.
+- *Infrastructure Recipes*: Swappable infrastructure that complies with organization best practicies and IT policy be default.
+- *Application Graph*: Understand how services and infrastructure in an application are interconnected.
+- *Cloud Neutral*: Deploy across development, on-premises and cloud environments with a consistent experience.
+- *Incremental Adoption*: Integrate Radius into existing workflows and existing catalogs of Infrastructure-as-Code templates.
 
 ## Release status
 
-Radius is currently in an invitation-only private release and we are working hard to get Radius ready for public release in the near future.
+This is an early release of Radius which enables the community to learn about and experiment with the platform. Please let us know what you think and open Issues when you find bugs or want to request a new feature. Radius is not yet ready for production workloads.
 
 ## Getting started
 
 1. Follow the [installation guide](https://docs.radapp.dev/getting-started/install/) to install Radius if you have not done so already.
-1. Then, follow the [first app guide](https://docs.radapp.dev/getting-started/first-app/) to take a tour of Radius by running your first app.
-
-## Quickstarts and samples
-
-See the [Radius quickstarts](https://docs.radapp.dev/getting-started/quickstarts/) and [samples repository](https://github.com/radius-project/samples) for tutorials and code examples that can help you get started with Radius.
+1. Visit the [Tutorials](https://docs.radapp.dev/tutorials) and [User Guides](https://docs.radapp.dev/guides) to get up and running with Radius.
```

Comment:
We should really point users to the first app tutorial explicitly as the step after installation. Maybe this should be 3 bullet points.

Reply:
```suggestion
1. Visit the [Tutorials](https://docs.radapp.dev/tutorials) and [User Guides](https://docs.radapp.dev/guides) to learn more about Radius and start radifying your apps
```

Getting started guide above is the first app experience. Sent a suggestion to make this a little clearer